### ISSUE TITLE
OpenStack installation: api service is not installed

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#182]
**version:** 618ba29

**steps to reproduce:**
```
[root@openstack ~]# systemctl status trinity_api
```
**actual output:**
```
trinity_api.service
   Loaded: not-found (Reason: No such file or directory)
   Active: inactive (dead)
```

**Comment by abhimukh** [comment]
I think, 461b752 fixes #181 and #182 not #183

**Comment by hansthen** [comment]
**Fixed in:** 4bcce3c
